check_test.go

```diff
@@ -226,16 +226,22 @@ var testcases = map[string]struct {
 	"CreateServiceOwnership": {
 		fixture: "check/create_service_ownership",
 		body: func(c *ol.Client) (*ol.Check, error) {
+			slackContactType := ol.ContactTypeSlack
 			return c.CreateCheckServiceOwnership(ol.CheckServiceOwnershipCreateInput{
-				CheckCreateInput: checkCreateInput,
+				CheckCreateInput:     checkCreateInput,
+				RequireContactMethod: ol.Bool(true),
+				ContactMethod:        &slackContactType,
```

Comment:
did you try
```
ContactMethod:        &ol.ContactTypeSlack,
```

would love to not have the variable but I understand why it might not be possible.